lib/improver/cli/nowcast_extrapolate.py

```diff
@@ -117,15 +118,21 @@ def main(argv=None):
         help="Desired units in which the accumulations should be expressed,"
         "e.g. mm")
 
+    # Load Cubes
     args = parser.parse_args(args=argv)
 
+    metadata_dict = load_json_or_none(args.json_file)
+
     upath, vpath = (args.eastward_advection_filepath,
                     args.northward_advection_filepath)
     spath, dpath = (args.advection_speed_filepath,
                     args.advection_direction_filepath)
 
     # load files and initialise advection plugin
     input_cube = load_cube(args.input_filepath)
+    oe_cube = load_cube_or_none(args.orographic_enhancement_filepaths)
+
+    scube = dcube = ucube = vcube = None
```

Comment:
I'd prefer slightly longer/more descriptive var names,
doesn't need to include cube if you want them shorter.

Reply:
oe, s and d have beeen lengthened. u and v are  known letters for wind velocity.